pennylane/measurements/measurements.py

```diff
@@ -419,6 +419,9 @@ def expand(self):
         if self.obs is None:
             raise qml.operation.DecompositionUndefinedError
 
+        if isinstance(self.obs, qml.ops.op_math.Prod):
+            self.obs = self.obs.terms()[1][0]
```

Comment:
This looks a bit drastic to do on this high level. Perhaps another data point that we should set the default for lazy arithmetic operations to be `False` cc @albi3ro 

What about cases where there is a non-trivial scalar coefficient? (e.g. `X(0) @ (0.5 * X(1))` would lose the `0.5`)

Reply:
Perhaps
```suggestion
        if isinstance(self.obs, qml.ops.op_math.Prod):
            self.obs = qml.simplify(self.obs)[0]
```

Reply:
For the moment I would suggest to remove the change in `measurements.py` and change `tests/transforms/test_qcut.py::TestCutCircuitWithHamiltonians::test_circuit_with_hamiltonian` from `qml.PauliZ(1) @ qml.PauliZ(2) @ qml.PauliZ(3)` >> `qml.prod(qml.Z(1), Z(2), @ qml.Z(3))` and add a warning to the docs of qcut that tensor products with more than 2 sites need to be put in a `prod`. This will be fixed by changing the default to `lazy=False`

Reply:
Done :)